packages/kit-bg/src/vaults/impls/tron/Vault.ts

```
@@ -861,4 +870,145 @@ export default class Vault extends VaultBase {
+          {
+            route: 'trxres',
+            params: {
+              method: 'post',
+              url: '/api/v1/order/create',
+              data: createOrderParams,
+              params: {},
+            },
+          },
+        ],
+      });
+    if (resp && resp[0] && resp[0].success !== false && !resp[0].error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)